### PR TITLE
Deadlock fixes from master

### PR DESCRIPTION
**Pull request by jherico**
Combined fixes for several potential spurious deadlocks / crashes:

#12368 
#12364 
#12370

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/7057/app_12373_PR.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/7057/HighFidelity-Beta-PR12373-7eecc4767274a3e1aeba90b5670f963a3f0d7453.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/7057/HighFidelity-Beta-PR12373-7eecc4767274a3e1aeba90b5670f963a3f0d7453.dmg

**Comment by sbennett77** [comment]
Currently Testing.

**Comment by sbennett77** [comment]
Passed.

@conklin94122

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -2404,7 +2403,9 @@ void Application::initializeUi() {
         tabletScriptingInterface->getTablet(SYSTEM_TABLET);
     }
     auto offscreenUi = DependencyManager::get<OffscreenUi>();
+    DeadlockWatchdogThread::pause();
```
Comment (ZappoMan) [inline interface/src/Application.cpp]: won't block merge, but why not use the `DeadlockWatchdogThread::withPause()` construct here?

Comment (jherico) [reply interface/src/Application.cpp]: These were two PRs originally and I didn't switch to the safer lambda approach till the second one.  I'll try to slip the withPause version into master in some other PR.

**Pull request merged**